swift-sdk/Internal/InternalIterableAPI.swift

```diff
@@ -706,7 +706,16 @@ final class InternalIterableAPI: NSObject, PushTrackerProtocol, AuthProvider {
         guard let launchOptions = launchOptions else {
             return
         }
+        
         if let remoteNotificationPayload = launchOptions[UIApplication.LaunchOptionsKey.remoteNotification] as? [AnyHashable: Any] {
+            
+            if let aps = remoteNotificationPayload[Const.RemoteNotification.aps] as? [String: Any],
+               let contentAvailable = aps[Const.RemoteNotification.contentAvailable] as? Int,
+               contentAvailable == 1 {
+                ITBInfo("Received push notification with wakey content-available flag")
+                return
```

Comment:
Looks like this can indeed stop the further calling of performing defaultNotification actions. But will it prevent any background processing the app needed to do?

Reply:
Also thinking if we return we will miss these events and if these events needed a dedicated tracking. Something like - background push delivery events?

Reply:
> will it prevent any background processing the app needed to do

Not really I think. The `handle(launchOptions:)`  only tracks the push default action and performs the default action — which should not happen on the background delivery of the push.

As to the second question, we can add that functionality afterwards. Not sure if we are tracking those on the other platforms. What do you think?

Reply:
@Ayyanchira gentle ping on this one!